src/gui/contacts/details.sh

```diff
@@ -6,7 +6,7 @@ gui_contacts_details(){
                 </tt></big>"
 
     [[ -z "$contact_id" ]] \
-    && message error "<tt>Please select a contact first.<tt>" \
+    && message error "<tt>Please select a contact first.</tt>" \
```

Comment:
I have used `<tt>` to show some message that usually appears on terminal, or when the font needs to be fixed-width (for better formatting). I am not sure whether this tag is needed in this case.